views/sections/faq.ejs

```diff
@@ -1,5 +1,5 @@
 <!-- CSS Customizations -->
-<link href="/public/faq-styles.css" rel="stylesheet">
+<link href="/public/css/faq-styles.css" rel="stylesheet">
```

Comment:
CSS should be contained in the `<head>` tag ideally.

Reply:
Alright alright you got me ;)